Singer.API/ClientApp/src/app/modules/admin/components/singerevents/singerevent-overview/singerevent-overview.component.html

```diff
@@ -127,6 +132,7 @@ <h5>{{ row.location.name }}</h5>
          mat-row
          *matRowDef="let row; columns: displayedColumns"
          class="selectable"
+         [ngClass]="{'maxRegistrantsExceeded': getMaxRegistrantsExceeded(row)}"
```

Comment:
This applies the `maxRegistrantsExceeded` to the whole row, we first said to only apply this to the cell/contents of the cell. Was this done deliberately or not?